### DISCUSSION TITLE
tine podcast

### DISCUSSION THREAD

**Discussion by pschuele** [tine20/tine20#7390] [General]
hey,
the first episode of the new tine podcast is online :)
https://www.tine-groupware.de/podcast/0-ist-das-mikro-an/ (in german)

it can also be found on spotify, itunes, deezer, ...
we also provice all episodes here: https://www.tine-groupware.de/tine-podcast/

looking forward to your feedback!

**Comment by ignoreigor** [comment]
Ich weiß ja nicht, ob ausgerechnet ein tine-Podcast gefehlt hat?! Aber ja: er hat gefehlt - weil er eine Kommunikationslücke füllt. Er wirkt sehr authentisch und vermittelt Einblicke in Euer Tun und Eure Ideen. 
Gerne ab und zu eine Episoden mit den Dingen, die Euch durch den Kopf gehen oder gerade im Fokus stehen!

**Reply by pschuele** [reply]
danke für dein feedback!

**Comment by lab-at-nohl** [comment]
Very nice, I like such insights. Please address your plans further (refactoring), also some architecture stuff would be nice (e.g. class loading or autowiring). Hidden gems were already discussed but I missed an explanation how to not remove things but bring it behind a feature switch... Go on!

**Reply by pschuele** [reply]
thank you for the feedback and suggestions!

**Comment by lab-at-nohl** [comment]
No. 1 was also interesting. You do have an executive director... Who had thought so.

Where can I find your docker config? While I am not planning to use docker, I have a full stack of openldap, cyrus-imapd, postfix, mariadb, php-fpm and apache. 

With eol of CentOS 7 or actually eol of the distro, I will switch back to my beloved openSUSE after ten years. However, I plan to replace e.g. openldap by 389-ds and are willing to replace Cyrus by dovecot, if my needs are still non commercially available (fulltext, feedback of logins to ldap). Maybe I could achieve this with Cyrus and notifyd, too. I am still in planning phase. What will you do?

**Reply by pschuele** [reply]
> Where can I find your docker config?

here is an example of a docker-compose file: https://github.com/tine20/tine20/blob/main/docs/docker/docker-compose.yml
you'll also find a quickstart-howto in the same directory.

> What will you do?

I'm not sure what you are referring to? :wink:

**Reply by lab-at-nohl** [reply]
> I'm not sure what you are referring to? wink

Sorry, I will be a little more expressive. In the podcast you explain that you will use docker instead of packages (rpm/deb) because it gets more complex to glue the stack of server components together, if I got you right... 

I wanted to know, which server components will you use or support equally in future. Because I am on the way to switch my system. Thus, I have the chances to make good choices! I ain't use docker, I want to setup the stack myself.

**Reply by pschuele** [reply]
ah I see!
sorry for the late reply - I was on vacation for a few days.

> In the podcast you explain that you will use docker instead of packages (rpm/deb) because it gets more complex to glue the stack of server components together, if I got you right...

yes, that is one of the pros for using docker. there are more - maybe we'll discuss it in more detail in a future episode :)

> I wanted to know, which server components will you use or support equally in future. Because I am on the way to switch my system. Thus, I have the chances to make good choices! I ain't use docker, I want to setup the stack myself.

- mysql or mariadb
- postfix+dovecot
- ldap or active directory
- redis
- clamav
- tika
- onlyoffice/documentserver
- tinegroupware/broadcasthub
- tinegroupware/document-preview-service

... maybe there are more

we'll update the sample docker-compose.yml and add the optional services soon.

**Comment by pschuele** [comment]
Here is the second episode (the first official one) -> https://www.tine-groupware.de/podcast/01-der-spagat/

**Comment by pschuele** [comment]
I'll open a new discussion for each podcast episode. thanks for the feedback so far ... and remember: please tell us the topics you are interested in and that we should discuss in one of the next episodes!
